ios/CodePush/CodePushConfig.m

```diff
@@ -34,7 +35,8 @@ - (instancetype)init
     NSString *buildVersion = [infoDictionary objectForKey:(NSString *)kCFBundleVersionKey];
     NSString *deploymentKey = [infoDictionary objectForKey:@"CodePushDeploymentKey"];
     NSString *serverURL = [infoDictionary objectForKey:@"CodePushServerURL"];
-
+    NSString *publicKey = [infoDictionary objectForKey:@"PublicKey"];
```

Comment:
As the value is taken from plist file that can be shared between different libraries does it make sense to name it `CodePushPublicKey` to avoid confusion about the reference of this property? (the same as for deploymentKey/ServerUrl)

Reply:
@max-mironov makes sense, I'll do it, tnx!